app/assets/stylesheets/govuk-component/_govspeak.scss

```diff
@@ -413,4 +400,103 @@
   .media-player {
     @include media-player;
   }
+
+  // This block is duplicated from Whitehall as a transitional step, see the
+  // commit adding this warning before making changes, original version:
+  // https://github.com/alphagov/whitehall/blob/master/app/assets/stylesheets/frontend/helpers/_attachment.scss
+  $thumbnail-width: 99px;
+
+  .attachment {
```

Comment:
I think the `.attachment` styles could be split out into a separate file and referenced here, much like the media player.